nmpc_cgmres/CMakeLists.txt

```diff
@@ -12,7 +12,6 @@ set(CXX_DISABLE_WERROR ON)
 
 option(INSTALL_DOCUMENTATION "Generate and install the documentation" OFF)
 
-include(../cmake/base.cmake)
 project(nmpc_cgmres LANGUAGES CXX)
```

Comment:
@ThomasDuvinage why don't you want to use `include(../cmake/base.cmake)` (submodule) anymore?

Reply:
@ThomasDuvinage I fixed error Ctest `No tests were found` after you remove submodule and implement CI colcon standalone in PR: https://github.com/chuoru/NMPC/pull/4
Please take a look when @chuoru  have time. Thanks !